pages/migration/delegator.tsx

```diff
@@ -58,47 +64,40 @@ export default function DelegateMigrationPage(): React.ReactElement {
       )}
 
       {account && (
-        <Stack gap={3}>
-          <Heading mb={2} as="h4" sx={{ textAlign: 'center' }}>
-            {(isDelegatedToExpiredContract || isDelegatedToExpiringContract) &&
-              'ACTION REQUIRED: Migrate your delegated MKR.'}
-
-            {!isDelegatedToExpiredContract &&
-              !isDelegatedToExpiringContract &&
-              "You don't need to migrate your MKR to a new delegate."}
-          </Heading>
-
-          {(isDelegatedToExpiredContract || isDelegatedToExpiringContract) && (
-            <Text
-              as="h3"
-              sx={{ textAlign: 'center', fontWeight: 'semiBold', maxWidth: '550px', margin: '0 auto' }}
-            >
-              One or more of your MakerDAO delegate&lsquo;s contracts are expiring.
-            </Text>
-          )}
-
+        <Stack gap={4} sx={{ maxWidth: '950px', margin: '0 auto' }}>
           <Box>
-            <Text as="p" variant="body">
-              Maker delegate contracts expire after 1 year.
-            </Text>
-            <Text>
-              Please migrate your MKR by undelegating from the expiring/expired contracts and redelegating to
-              the new contracts.
+            <Heading mb={2} as="h4" sx={{ textAlign: 'left' }}>
+              Action required: Migrate your delegated MKR.
+            </Heading>
+            <Text as="p" sx={{ color: 'onSecondary' }}>
+              One or more of your MakerDAO delegate&lsquo;s contracts are expiring.
             </Text>
           </Box>
 
-          <Box>
-            <Text>
-              Please find below any expiring/expired delegate contracts you delegated MKR to, and requires
-              migration.
-            </Text>
-          </Box>
+          <Card sx={{ display: 'flex', alignItems: 'flex-start' }}>
+            <Box sx={{ mr: 2, mt: 1 }}>
+              <Icon name="info" />
+            </Box>
+            <Box>
+              <Text as="p" sx={{ fontWeight: 'semiBold' }}>
+                Maker delegate contracts expire after 1 year.
+              </Text>
+              <Text as="p" sx={{ color: 'onSecondary' }}>
+                Please migrate your MKR by undelegating from the expiring/expired contracts and redelegating
+                to the new contracts.
+              </Text>
+              <Text as="p" sx={{ color: 'onSecondary' }}>
+                Find in this page any expiring/expired delegate contracts you delegated MKR to, and requires
+                migration.
+              </Text>
+            </Box>
```

Comment:
IMO this should be some sort of reusable Alert or something that we already have. Maybe we can check with Tiago.